### PR TITLE
feat: full feature parity

### PR DESCRIPTION
**Pull request by peterdeme**
- full feautre parity

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -28,6 +28,6 @@ jobs:
 
       - run: bundle exec rake test
         env:
-          STREAM_CHAT_API_KEY: ${{ secrets.STREAM_CHAT_API_KEY }}
-          STREAM_CHAT_API_SECRET: ${{ secrets.STREAM_CHAT_API_SECRET }}
+          STREAM_KEY: ${{ secrets.STREAM_CHAT_API_KEY }}
+          STREAM_SECRET: ${{ secrets.STREAM_CHAT_API_SECRET }}
```
Comment (ffenix113) [inline .github/workflows/ci.yml:32]: Can you please also check `STREAM_CHAT_URL` vs `STREAM_CHAT_API_HOST` usage in code?

**Comment by ffenix113** [comment]
Amazing work!

File: .github/workflows/ci.yml (hunk 1 above)
Comment (peterdeme) [reply .github/workflows/ci.yml:32]: @ffenix113
image

Comment (ffenix113) [reply .github/workflows/ci.yml:32]: Thanks, but `STREAM_CHAT_API_HOST` is used in CI configuration and tests, while `STREAM_CHAT_URL` is only used when creating client. Could you please check which one should be used everywhere?

Comment (peterdeme) [reply .github/workflows/ci.yml:32]: Sure. I attached a picture of the ones that should be used. I removed `STREAM_CHAT_API_HOST` from the CI, it doesn't make sense for us. I've checked the usages.

image

File: spec/client_spec.rb [hunk 2]
```diff
@@ -20,7 +20,7 @@ def loop_times(times)
   end
 
   before(:all) do
-    @client = StreamChat::Client.new(ENV['STREAM_CHAT_API_KEY'], ENV['STREAM_CHAT_API_SECRET'], base_url: ENV['STREAM_CHAT_API_HOST'])
+    @client = StreamChat::Client.new(ENV['STREAM_KEY'], ENV['STREAM_SECRET'], base_url: ENV['STREAM_CHAT_API_HOST'])
```
Comment (ffenix113) [inline spec/client_spec.rb:23]:
```suggestion
    @client = StreamChat::Client.new(ENV['STREAM_KEY'], ENV['STREAM_SECRET'], base_url: ENV['STREAM_CHAT_URL'])
```

File: spec/channel_spec.rb [hunk 3]
```diff
@@ -19,7 +19,7 @@ def loop_times(times)
   end
 
   before(:all) do
-    @client = StreamChat::Client.new(ENV['STREAM_CHAT_API_KEY'], ENV['STREAM_CHAT_API_SECRET'], base_url: ENV['STREAM_CHAT_API_HOST'])
+    @client = StreamChat::Client.new(ENV['STREAM_KEY'], ENV['STREAM_SECRET'], base_url: ENV['STREAM_CHAT_API_HOST'])
```
Comment (ffenix113) [inline spec/channel_spec.rb:22]:
```suggestion
    @client = StreamChat::Client.new(ENV['STREAM_KEY'], ENV['STREAM_SECRET'], base_url: ENV['STREAM_CHAT_URL'])
```

File: .github/workflows/ci.yml (hunk 1 above)
Comment (ffenix113) [reply .github/workflows/ci.yml:32]: Thanks. Added two comments(maybe forgot to push).

When they are done - let's merge it.

File: spec/client_spec.rb (hunk 2 above)
Comment (peterdeme) [reply spec/client_spec.rb]: ✅

File: spec/channel_spec.rb (hunk 3 above)
Comment (peterdeme) [reply spec/channel_spec.rb]: ✅

**Pull request merged**